src/ws-handler.ts

```diff
@@ -165,6 +168,7 @@ export class WsHandler {
                 this.server.metricsManager.markDisconnection(ws);
             }
         });
+        this.clearTimeout(ws);
```

Comment:
This should not exist here. It should be placed in the `unsubscribeFromAllChannels()`'s `.then()` closure, so we make sure this gets cleared after the close.
Paradoxically, as long as the `ws` connection gets completely memory-deleted on close, i still think that Node.js garbage collector should have less to collect if we can do it manually.